app/views/catalog/_part_header_default.html.erb

```diff
@@ -2,7 +2,7 @@
 <div class="part-header">
   <h3><%= show_presenter.field_value('ms_locus_tesim') %></h3>
   <p>
-    <span class="part-author"><%= show_presenter.field_value('ms_author_tesim') %></span>
+    <span class="part-author"><%= show_presenter.field_value('ms_author_header_tesim') %></span>
```

Comment:
I'm not super happy about adding this additional field, but it seemed like a more future-proof solution than a helper that might break once Vatican updates their TEI. 

## Before
![screen shot 2018-12-17 at 4 06 30 pm](https://user-images.githubusercontent.com/5402927/50123550-110b4c80-0216-11e9-8d9e-664442ac543d.png)

## After
![screen shot 2018-12-17 at 4 06 35 pm](https://user-images.githubusercontent.com/5402927/50123558-16689700-0216-11e9-967b-c1811151a97a.png)